JSX.md

```diff
@@ -58,19 +54,25 @@ The `as` operator is available in both `.ts` and `.tsx` files.
 
 # Type Checking
 
-In order to understand type checking with JSX you must first understand intrinsic elements vs value-based elements. Given a JSX expression `<expr />`, is `expr` referring to something intrinsic to the environment (ie, a div or span in a DOM environment) or to a custom component that you've made? This is important for two reasons:
+In order to understand type checking with JSX you must first understand the difference between intrinsic elements value-based elements.
+Given a JSX expression `<expr />`, `expr` may either refer to something intrinsic to the environment (e.g. a `div` or `span` in a DOM environment) or to a custom component that you've created.
+This is important for two reasons:
 
-1. For React, intrinsic elements are emitted as strings, like `React.createElement('div')`, whereas a component you've created is not: `React.createElement(MyComponent)`.
+1. For React, intrinsic elements are emitted as strings (`React.createElement("div")`), whereas a component you've created is not (`React.createElement(MyComponent)`).
 2. The types of the attributes being passed in the JSX element should be looked up differently. Intrinsic element attributes should be known *intrinsically* whereas components will likely want to specify their own set of attributes.
```

Comment:
Perhaps the clearest way of making the distinction is that non-intrinsic elements are resolved to a value in scope, as if they were a variable, whereas intrinsic elements don't correspond to any runtime variable. It's somewhat like `$('div')` vs `$(myElem)`.